iwyu_path_util.cc

```diff
@@ -82,9 +82,11 @@ string Basename(const string& path) {
 }
 
 string GetCanonicalName(string file_path) {
-  // Get rid of any <> and "" in case file_path is really an #include line.
-  StripLeft(&file_path, "\"") || StripLeft(&file_path, "<");
-  StripRight(&file_path, "\"") || StripRight(&file_path, ">");
+  // For this special 'path' we just remove the quotes.
+  if (file_path == "<built-in>")
+    return "built-in";
```

Comment:
Good catch, I just noticed this too. Can you just return `file_path` instead, so we maintain `<built-in>`?